look-and-feel/react/src/Link/Link.tsx

```diff
@@ -1,31 +1,41 @@
-import { ComponentPropsWithoutRef, PropsWithChildren } from "react";
-import "@axa-fr/design-system-look-and-feel-css/dist/Link/Link.scss";
 import openInNew from "@material-symbols/svg-400/outlined/open_in_new.svg";
+import {
+  ComponentPropsWithoutRef,
+  PropsWithChildren,
+  type ReactNode,
+} from "react";
 import { Svg } from "../Svg";
 
 type LinkProps = {
   openInNewTab?: boolean;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
 } & ComponentPropsWithoutRef<"a">;
 
 export const Link = ({
   href,
-  openInNewTab,
+  openInNewTab = false,
+  leftIcon,
+  rightIcon,
   children,
   ...props
 }: PropsWithChildren<LinkProps>) => {
   const newTabProps = openInNewTab && {
     target: "_blank",
     rel: "noopener noreferrer",
   };
+
   return (
     <a
       className={`af-link ${openInNewTab ? " af-link--openInNewTab" : ""}`}
```

Comment:
```suggestion
      className={`af-link${openInNewTab ? " af-link--openInNewTab" : ""}`}
```

Reply:
C'est normal, c'est pour éviter d'avoir ça dans le nom de la classe : <a class="af-link ">
L'espace est rajouté lorsque la classModifier est ajouté

Reply:
Fixed in : https://github.com/AxaFrance/design-system/pull/587/commits/017dd41188fbb2153d6938164325612c864f8f9f